api/organisations/chargebee/chargebee.py

```diff
@@ -102,6 +102,10 @@ def get_hosted_page_url_for_subscription_upgrade(
 def get_subscription_metadata(
     subscription_id: str,
 ) -> typing.Optional[ChargebeeObjMetadata]:
+    if not (subscription_id and subscription_id.strip() != ""):
```

Comment:
I'm not entirely sure what this is doing differently to the code below? Since we're suppressing the error it will just return `None` anyway, right? Sounds we return the default `ChargebeeObjMetadata()` instead? Where exactly is the error coming from?

Reply:
It's because chargebee raises Exception instead of ChargebeeAPIError(the one we are catching)
if subscription ID is none or empty string. Returning the default happens in subscription/service method that uses the name.